sed/config/mpes_example_config.yaml

```diff
@@ -79,6 +73,25 @@ dataframe:
     kx: '1/A'
     ky: '1/A'
 
+  # dataframe channels and group names to read from the h5 files
+  channels:
+    # The timestamp
```

Comment:
i understand your timestamps come from the msmarkers group so what does this comment mean here?
Also, the ms_markers_group doesn't seem to be defined in the config but is hardcoded as default values?

Reply:
This is a copy paste relict, fixed. 
The ms_marker_group is defined outside the channels, because it does need a separate mechanism. Maybe I also change this, but this also performs rather poorly, so I want to keep the option of only generating them if necessary.